src/Presentation/RestApi/Controller/v1/InitController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Presentation\RestApi\Controller\v1;
+
+use SprykerSdk\Sdk\Infrastructure\Service\Initializer;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class InitController
+{
+    /**
+     * @var \SprykerSdk\Sdk\Infrastructure\Service\Initializer
+     */
+    protected Initializer $initializerService;
+
+    /**
+     * @param \SprykerSdk\Sdk\Infrastructure\Service\Initializer $initializerService
+     */
+    public function __construct(Initializer $initializerService)
+    {
+        $this->initializerService = $initializerService;
+    }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function __invoke(Request $request): Response
+    {
+        $result = $this->initializerService->initialize($request->request->all());
+
+        return new JsonResponse(['result' => $result ? 'OK' : 'FAILED']);
```

Comment:
Can we somehow to add failed response message text?

Reply:
Failure will be a global event.